com.unity.ml-agents/Runtime/Sensor/WriteAdapter.cs

```diff
@@ -24,7 +24,7 @@ public class WriteAdapter
         /// <param name="data">Float array or list that will be written to.</param>
         /// <param name="shape">Shape of the observations to be written.</param>
         /// <param name="offset">Offset from the start of the float data to write to.</param>
-        public void SetTarget(IList<float> data, int[] shape, int offset)
+        internal void SetTarget(IList<float> data, int[] shape, int offset)
```

Comment:
I think we need to keep this class public; it would be impossible to test a custom sensor without these methods.